client/sites-dashboard/components/sites-table.tsx

```diff
@@ -115,36 +130,39 @@ export function SitesTable( { className, sites, isLoading = false }: SitesTableP
 	}, [ masterbarHeight ] );
 
 	return (
-		<Table className={ className }>
-			<THead
-				blockOffset={ masterbarHeight }
-				ref={ headerRef }
-				style={ isHeaderStuck ? headerShadow : undefined }
-			>
-				<Row>
-					<th style={ { width: '50%' } }>{ __( 'Site' ) }</th>
-					<th style={ { width: '20%' } }>{ __( 'Plan' ) }</th>
-					<th>{ __( 'Last Publish' ) }</th>
-					<th>{ __( 'Status' ) }</th>
-					<th style={ { width: '24px' } }></th>
-				</Row>
-			</THead>
-			<tbody>
-				{ isLoading &&
-					Array( N_LOADING_ROWS )
-						.fill( null )
-						.map( ( _, i ) => (
-							<SitesTableRowLoading
-								key={ i }
-								columns={ 5 }
-								delayMS={ i * 150 }
-								logoProps={ { width: 108, height: 78 } }
-							/>
-						) ) }
-				{ sites.map( ( site ) => (
-					<SitesTableRow site={ site } key={ site.ID }></SitesTableRow>
-				) ) }
-			</tbody>
-		</Table>
+		<>
+			<Table className={ className }>
+				<THead
+					blockOffset={ masterbarHeight }
+					ref={ headerRef }
+					style={ isHeaderStuck ? headerShadow : undefined }
+				>
+					<Row>
+						<th style={ { width: '50%' } }>{ __( 'Site' ) }</th>
+						<th style={ { width: '20%' } }>{ __( 'Plan' ) }</th>
+						<th>{ __( 'Last Publish' ) }</th>
+						<th>{ __( 'Status' ) }</th>
+						<th style={ { width: '24px' } }></th>
+					</Row>
+				</THead>
+				<tbody>
+					{ isLoading &&
+						Array( N_LOADING_ROWS )
+							.fill( null )
+							.map( ( _, i ) => (
+								<SitesTableRowLoading
+									key={ i }
+									columns={ 5 }
+									delayMS={ i * 150 }
+									logoProps={ { width: 108, height: 78 } }
+								/>
+							) ) }
+					{ sites.map( ( site ) => (
+						<SitesTableRow site={ site } key={ site.ID }></SitesTableRow>
+					) ) }
+				</tbody>
+			</Table>
+			<LinkInBioBanner sites={ sites } displayMode={ 'row' } />
```

Comment:
We can directly pass a string through props.

```suggestion
			<LinkInBioBanner sites={ sites } displayMode="row" />
```